civihr_default_theme/scss/base/_variables.scss

```diff
@@ -1,5 +1,9 @@
 @import url('//fonts.googleapis.com/css?family=Lato:400,300,300italic,400italic,700,700italic');
 
+
+$chr-gradient-1: linear-gradient(315deg, #01A2D5 0%, #0070BB 100%);
```

Comment:
shouldn't the start/end colors of the gradients part of the palette (= be one of the color variables)

Reply:
They are not used anywhere else, so I did not move them to variables. Should I?